lib/register/transformGroup.js

```diff
@@ -27,10 +27,10 @@ function registerTransformGroup(options) {
   if (!_.isArray(options.transforms))
     throw new Error('transforms must be an array of registered value transforms');
 
-  options.transforms.forEach(function(t) {
+  options.transforms.forEach((function(t) {
     if (_.has(this.transforms, t))
```

Comment:
Not your fault, but looking at this I discovered this code is actually broken. this.transforms is undefined, the real thing we need is this.transform. And the conditional is backwards, we want to make sure that each transform matches a transform already defined. If we revert this code you added, this error will never throw so it was never catching what it was supposed to!

So lets change this line to
`if (!_.has(this.transform, t))`

We will need to update line 117 in test/registerTransformGroup.js to use an actual prebuilt transform like 'color/rgb' or 'name/cti' as this test will now fail because 'foo' is not a registered transform.